lib/contracts/l1bridge/deploy.go

```diff
@@ -68,9 +68,26 @@ func getDeployCfg(network netconf.ID) (DeploymentConfig, error) {
 		return stagingCfg(), nil
 	}
 
+	if network == netconf.Omega {
+		return omegaCfg(), nil
+	}
+
 	return DeploymentConfig{}, errors.New("unsupported network", "network", network)
 }
 
+func omegaCfg() DeploymentConfig {
```

Comment:
Can we just have a static `map[netconf.ID]DeploymentConfig`? That will make a lot of this cleaner (fewer methods, no switching on the network ID in `getDeployCfg` above, etc.).

Reply:
Yeah think that's probably cleaner. 

These funcs (seen elsewhere in lib/contracts) were initially used to avoid global vars, cause linter didn't like. But we've removed that rule and use global vars in a number of places. I can clean up and introduce in a separate pr